docs/kpod-inspect.1.md

```diff
@@ -0,0 +1,36 @@
+## kpod-inspect "1" "July 2017" "kpod"
+
+## NAME
+kpod inspect - display a container or image's configuration
+
+## SYNOPSIS
+**kpod** **inspect** [*options* [...]] name
+
+## DESCRIPTION
+Displays the configuration of a container or image
```

Comment:
docker inspect man page has
```
DESCRIPTION
       This  displays  all  the information available in Docker for a given container or image. By default, this will render all results in a JSON array. If the container and image have the same name, this
       will return container JSON for unspecified type. If a format is specified, the given template will be executed for each result.
```